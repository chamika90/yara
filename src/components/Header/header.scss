@import "../../resources/colors.scss";
.container {
    width: 100%;
    //height: 80%;
    padding: 2px;
    //background-color: $gray3;
    //align-items: flex-start;
}

.title {
    //background-color: antiquewhite;
    text-align: left;
}