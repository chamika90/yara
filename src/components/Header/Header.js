import React from "react";
import "./header.scss";

export default function Header() {
  return (
    <div className="container">
      <h1 className="title">Environmental Measurements Across Germany</h1>
    </div>
  );
}