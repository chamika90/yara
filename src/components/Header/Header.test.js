import { render } from "@testing-library/react";
import Header from "./Header";

describe('render header', () => {
    it('Should match snapshot', () => {
        const {toJSON}  = render(<Header/>);
        expect(toJSON).toMatchSnapshot();
    });
});