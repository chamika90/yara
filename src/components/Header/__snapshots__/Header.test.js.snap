// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header Should match snapshot 1`] = `undefined`;
